editor/src/clj/editor/app_view.clj

```diff
@@ -1906,14 +1906,19 @@ If you do not specifically require different script states, consider changing th
          (log/error :exception e)
          nil)))
 
+(defn- merge-keymaps [prefs]
+  (let [custom-keymap (or (open-custom-keymap (prefs/get-prefs prefs "custom-keymap-path" "")) [])
+        default-keymap keymap/default-host-key-bindings
+        merged-keymap (concat custom-keymap default-keymap)]
```

Comment:
Not sure if this is the right way? The resulting keymap will contain duplicates if the custom and default has the same keybinding, but since the custom bindings are first they will be the ones used. It works but should we remove duplicates?